core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2679,7 +2685,7 @@ class SparkContext(config: SparkConf) extends Logging {
  * various Spark features.
  */
 object SparkContext extends Logging {
-  private val VALID_LOG_LEVELS =
+  val VALID_LOG_LEVELS =
```

Comment:
Given the use case, the following will be sufficient.
```scala
-  val VALID_LOG_LEVELS =
+  private[spark] val VALID_LOG_LEVELS =
```

Reply:
Thanks! It's now updated.